src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/RootControllerTest.java

```diff
@@ -1,24 +0,0 @@
-package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
-import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.test.web.servlet.MockMvc;
-
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
-@RunWith(SpringRunner.class)
-@WebMvcTest
-public class RootControllerTest {
-
-    @Autowired
-    private transient MockMvc mockMvc;
-
-    @Test
-    public void call_to_root_endpoint_should_result_with_204_response() throws Exception {
```

Comment:
any reason we removed this test ?

Reply:
because it is an integration test and it exists in integration test suite matching char by char the whole file :D.

p.s. i did include as a separate commit here https://github.com/hmcts/bulk-scan-orchestrator/pull/48/commits/ff0276ec72cd0f743a9ded7d8e7a33b79826e379.

noticed as unit tests took 3mins to run getting stuck on this test. and integration tests have only that one too